styles.css

```diff
@@ -0,0 +1,374 @@
+    overflow-y: scroll;
+}
+.head-card-item{
+    height: 80%;
+    min-width: 18rem;
+    margin: 0.2rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    border-radius: 3rem;
+    position: relative;
+}
+.head-card-item.active img{
+    border: 5px solid #CD3D0F;
+}
+.head-card-item img{
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 1rem;
+    filter: brightness(1);
+    z-index: 0;
+    transition: 0.3s;
+}
+.head-card-item:hover img{
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 1rem;
+    filter: brightness(0.3);
+    z-index: 0;
+}
+
+.head-card-item:hover span{
+    position: absolute;
+    color: white;
+    font-size: 1.2rem;
+    font-weight: 500;
+    z-index: 1;
+}
+.head-cards::-webkit-scrollbar{
+    display: none;
+}
+
+.mid-container{
+    background-image: linear-gradient(black, purple);
+    height: 45rem;
+    width: 100%;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.mid-header{
+    width: 80%;
+    text-align: center;
+    
+    padding: 1rem;
+    font-size: 1.5rem;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.pdt-cards{
+    width: 80%;
+    margin-top:5rem;
+    display: flex;
+    gap: 1rem;
+    background: transparent;
+    flex-direction: row;
+    overflow-y:hidden;
+}
+.card-item.horizontal{
+    min-width: 20rem;
+    min-height: 20rem;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    background-color: #af4261;
+    box-shadow: 0.5rem 0.5rem 0.5rem black; 
+}
+.card-item.horizontal .card-info{
+    display: flex;
+    flex-direction: column;
+}
+.card-item.horizontal .card-info .card-infoText{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    padding: 0rem 1rem 1rem 1rem;
+}
+.card-item.horizontal .card-info .card-infoText .card-content{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    
+}
+.card-item.horizontal .card-info .card-infoText .card-content button{
+    all: unset;
+    margin-left: 0.5rem;
+    padding: 0.5rem;
+    text-transform: uppercase;
+    background-color: purple;
+    border-radius: 1rem;
+    box-shadow: inset;
+    text-shadow: 1px 1px 1px black;
+
+}
+.card-item.horizontal .card-info .card-infoText .card-content span{
+    font-size: 1.5rem;
+}
+.card-item.horizontal .card-info .card-image img{
+
+    max-inline-size: 100%;
+    block-size: auto;
+    aspect-ratio: 4/3;
+    object-fit: contain/cover;
+    object-position: top center;
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+}
+.card-item.horizontal .card-buttons{
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: row;
```

Comment:
Its default right? Correct me if I am wrong

Reply:
Didn't get you there

Reply:
I meant like display:flex default direction is row right ?